packages/service/ni_measurement_plugin_sdk_service/discovery/_types.py

```diff
@@ -19,3 +21,10 @@ def insecure_address(self) -> str:
     def ssl_authenticated_address(self) -> str:
         """Get the service's SSL-authenticated address in the format host:port."""
         return f"{self.location}:{self.ssl_authenticated_port}"
+
+
+class ResolveServiceWithInformationResponse(typing.NamedTuple):
+    """Represents the location of a service along with its information."""
```

Comment:
I think this way it would emphasize that this class is more of an extension of the `ServiceInfo` class.
```suggestion
    """Represents the service information along with its location."""
```